### DISCUSSION TITLE
When creating a Local Volume, check the path for writability.

### DISCUSSION THREAD

**Discussion by andris-sevcenko** [craftcms/cms#8398] [Ideas]
### Description

So that you know immediately if something is wrong when attempting to save a Volume.

**Comment by andris-sevcenko** [comment]
Closing this as there are uses cases when you would want a read-only local volume and external volumes can be read-only as well, depending on the permissions.

**Comment by brandonkelly** [comment]
If that’s true, then Craft should have the concept of read-only volumes (#2052). Once that is implemented, this FR should be implemented as well, for writeable volumes.
